packages/office-ui-fabric-react/src/demo/components/DemoPage.types.ts

```diff
@@ -0,0 +1,45 @@
+import { IComponentStatusProps } from 'office-ui-fabric-react/lib/demo/ComponentStatus/ComponentStatus.types';
+
+export interface IDemoPageProps {
+  /** title that goes into the header */
+  title: string;
+
+  /** name of the component being documented */
```

Comment:
I know all these are not making it today, to the documentation page, but ultimately contributors to the OUFR library will be interacting with it when they fill out the IDemoPageProps object in each of the example page they craft.

So we really should have the same quality bar in our comments as we expect the directly public-facing documentation that makes it to the website.

You know.. broken window theory...

So `Name of the component being documented.` instead of `name of the component being documented`.

Reply:
I think we might want to write down the bar for documentation. I hadn't found a wiki page / readme on it so far. Perhaps I'll add it once I get this checked in.